content/en/docs/components/pipelines/pipelines-quickstart.md

```diff
@@ -92,65 +56,27 @@ workload:
     alt="Run results on the pipelines UI"
     class="mt-3 mb-3 border border-info rounded">
 
-You can find [the source code for the basic parallel join sample](https://github.com/kubeflow/pipelines/blob/master/samples/core/parallel_join/parallel_join.py)
-in the Kubeflow Pipelines repo.
+You can find the source code for the **Data passing in python components** tutorial
```

Comment:
It's better to use link text that describes the content that we are linking to. Suggested revision

You can find the [source code for the **Data passing in python components** tutorial](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components) in the Kubeflow Pipelines repo.